client/src/components/BackupCodes/Register.js

```diff
@@ -204,6 +205,7 @@ class Register extends Component {
 
 Register.propTypes = {
   codes: PropTypes.arrayOf(PropTypes.string),
+  copyFeedbackDuration: PropTypes.number,
```

Comment:
Do you think we should name this prop `copyFeedbackDurationMs` to indicate its value is in milliseconds? or maybe just add a comment above this line saying that it is?